tests/test_accuracy.py

```diff
@@ -16,7 +16,7 @@
 }
 
 def emulate(arch, binary, use_sim_procs, steps, hit_addrs, finished):
-    p = angr.Project(os.path.join(test_location, arch, binary), use_sim_procedures=use_sim_procs, rebase_granularity=0x1000000, load_debug_info=False)
+    p = angr.Project(os.path.join(test_location, arch, binary), use_sim_procedures=use_sim_procs, rebase_granularity=0x1000000, load_debug_info=False, auto_load_libs=True)# auto_load_libs can't be disabled as the test takes longer time to execute
```

Comment:
This line is way too long. Let's reformat it so each parameter to `Project` is on its own line. Also, `#` should always have spaces around it.